journal-of-forensic-sciences.csl

```diff
@@ -27,51 +28,47 @@
     </names>
   </macro>
   <macro name="editor">
-    <names variable="editor" suffix=".  ">
+    <names variable="editor">
       <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always"/>
       <label form="long" prefix=", "/>
     </names>
   </macro>
   <macro name="publisher">
-    <text variable="publisher-place" suffix=": "/>
-    <text variable="publisher" suffix=", "/>
-    <group suffix="; ">
+    <group delimiter=", ">
+      <group delimiter=": ">
+        <text variable="publisher-place"/>
+        <text variable="publisher"/>
+      </group>
       <date variable="issued">
         <date-part name="year"/>
       </date>
-      <text macro="accessed-date" prefix=" "/>
     </group>
   </macro>
   <macro name="access">
-    <group>
-      <text value="Available from: "/>
-      <text variable="URL"/>
-    </group>
-  </macro>
-  <macro name="accessed-date">
     <choose>
-      <if variable="URL">
-        <group prefix="[" suffix="]" delimiter=" ">
-          <text term="cited"/>
-          <date variable="accessed">
-            <date-part name="year"/>
-            <date-part name="month" prefix=" " form="short" strip-periods="true"/>
-            <date-part name="day" prefix=" "/>
-          </date>
-        </group>
+      <if type="article-journal" match="any">
+        <choose>
+          <if match="any" variable="volume page">
```

Comment:
I think you mean match="none" here? But reading quickly, so letting you double-check

Reply:
Thank you for spotting that! :)